src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -976,12 +968,12 @@
     <comment>Button label that deletes the unfocused appearance for this profile.</comment>
   </data>
   <data name="Profile_UseAcrylic.Header" xml:space="preserve">
-    <value>Enable acrylic</value>
-    <comment>Header for a control to toggle the acrylic-like rendering of the background. The acrylic material creates a translucent texture.</comment>
+    <value>Enable acrylic material</value>
+    <comment>Header for a control to toggle the use of acrylic material for the background. "Acrylic material" is a Microsoft-specific term.</comment>

```

Comment:
idk how localization works, exactly. But would it be helpful to link the doc you mentioned earlier? That way it's consistent?

Reply:
(same goes for the `Globals_AcrylicTabRow` entry above)

Reply:
I've added links to the en-us docs.